mongodbatlas/mongodbatlas.go

```diff
@@ -46,6 +46,7 @@ type Client struct {
 	PrivateIPMode                    PrivateIpModeService
 	MaintenanceWindows               MaintenanceWindowsService
 	Teams                            TeamsService
+	Auditings                        AuditingsService
```

Comment:
Auditing should be singular so no s, i.e. not Auditings.

Reply:
Okay, thank you so much I will change it 👍